src/index.js

```diff
@@ -1,10 +1,17 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import { Provider } from "react-redux";
 import "./index.css";
 import App from "./App";
 import * as serviceWorker from "./serviceWorker";
+import store from "./redux/store";
 
-ReactDOM.render(<App />, document.getElementById("root"));
+ReactDOM.render(
```

Comment:
This implementation will later cause problems. Kindly do the routing in the App.jsx file. This file should just remain as before.